x-pack/test/cases_api_integration/common/lib/utils.ts

```diff
@@ -1216,3 +1216,20 @@ export const createCaseAndBulkCreateAttachments = async ({
 
   return { theCase: patchedCase, attachments };
 };
+
+export const delay = (ms: number) => new Promise((resolve) => setTimeout(resolve, ms));
+
+export const calculateDuration = (closedAt: string | null, createdAt: string | null): number => {
+  if (closedAt == null || createdAt == null) {
+    throw new Error('Dates are null');
+  }
+
+  const createdAtMillis = new Date(createdAt).getTime();
+  const closedAtMillis = new Date(closedAt).getTime();
+
+  if (isNaN(createdAtMillis) || isNaN(closedAtMillis)) {
```

Comment:
Is it worth adding an extra check in here for `closedAtMillis < createdAtMillis`?

Reply:
Ok!